### ISSUE TITLE
managesf does not support project group

### ISSUE THREAD

**Issue by goneri** [softwarefactory-project/managesf#1]
managesf does not support project-group like this: https://github.com/ansible/project-config/blob/master/zuul/tenants.yaml#L107-L150
We can still flatten the list, this is not critical thus.

**Comment by TristanCacqueray** [comment]
Agreed that is unfortunate, we started defining the schema in dhall here: https://softwarefactory-project.io/cgit/software-factory/dhall-software-factory/tree/README.md . Though it seems like the source repositories options are incomplete.
